doc/metadata_routing.rst

```diff
@@ -9,42 +9,59 @@ Metadata Routing
     >>> sklearn.set_config(enable_metadata_routing=True)
 
-This guide demonstrates how metadata such as ``sample_weight`` can be routed
-and passed along to estimators, scorers, and CV splitters through
-meta-estimators such as :class:`~pipeline.Pipeline` and
-:class:`~model_selection.GridSearchCV`. In order to pass metadata to a method
-such as ``fit`` or ``score``, the object consuming the metadata, must *request*
-it. For estimators and splitters, this is done via ``set_*_request`` methods,
-e.g. ``set_fit_request(...)``, and for scorers this is done via the
-``set_score_request`` method. For grouped splitters such as
-:class:`~model_selection.GroupKFold`, a ``groups`` parameter is requested by
-default. This is best demonstrated by the following examples.
-
-If you are developing a scikit-learn compatible estimator or meta-estimator,
-you can check our related developer guide:
-:ref:`sphx_glr_auto_examples_miscellaneous_plot_metadata_routing.py`.
-
-.. note::
   Note that the methods and requirements introduced in this document are only
   relevant if you want to pass :term:`metadata` (e.g. ``sample_weight``) to a method.
   If you're only passing ``X`` and ``y`` and no other parameter / metadata to
-  methods such as :term:`fit`, :term:`transform`, etc, then you don't need to set
+  methods such as :term:`fit`, :term:`transform`, etc., then you don't need to set
   anything.
 
+This guide demonstrates how :term:`metadata` can be routed and passed between objects in
+scikit-learn. If you are developing a scikit-learn compatible estimator or
+meta-estimator, you can check our related developer guide:
+:ref:`sphx_glr_auto_examples_miscellaneous_plot_metadata_routing.py`.
+
+Metadata is data that an estimator, scorer, or CV splitter takes into account if the
+user explicitly passes it as a parameter. For instance, some methods in certain objects
+can take into account `sample_weight`, `classes`  or `groups` if provided by the user.
+Prior to scikit-learn version 1.3, there was no single API for passing metadata like
+that if these objects were used in conjunction with other objects, e.g. a scorer
+accepting `sample_weight` inside a :class:`~model_selection.GridSearchCV`.
+
+With the Metadata Routing API, we can transfer metadata to estimators, scorers, and CV
+splitters using :term:`meta-estimators` (such as :class:`~pipeline.Pipeline` or
+:class:`~model_selection.GridSearchCV`) or routing functions such as
+:func:`~model_selection.cross_validate`. In order to pass metadata to a method like
+``fit`` or ``score``, the object consuming the metadata, must *request* it. This is done
+via `set_{method}_request()` methods, where `{method}` is substituted by the name of the
+method that request the metadata. For instance, estimators and splitters, that use the
+metadata in their `fit()` method would use `set_fit_request()`, and scorers would use
+the `set_score_request()`. These methods allow us to specify which metadata to request,
```

Comment:
```suggestion
the `set_score_request()`. These methods allow you to specify which metadata to request,
```

Reply:
If we use "we" then we also need to use "us".